sponsor.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "XCP-NG SPONSOR"
+comments: false
+---
+XCP-ng sponsors span all industry sectors. They are trusting open source technologies to help them in their everyday business.
+
+## PLATINUM SPONSORS
```

Comment:
IMHO, it's better to use titles in lower case: "Platinum sponsors"

Reply:
ok - changed